app/src/main/java/comp3350/mealbuddy/objects/Edible.java

```diff
@@ -0,0 +1,24 @@
+package comp3350.mealbuddy.objects;
+
+
+import java.util.List;
+import java.util.Map;
+
+import comp3350.mealbuddy.objects.MacroFactory.Macros;
+import comp3350.mealbuddy.objects.MicroFactory.Micros;
+
+public abstract class Edible {
+
+    public String name;
+    public List<String> labels;
+    public Map<Micros, Integer> micros;
```

Comment:
Didnt bother making accessors/getters since this is just a example that might get thrown away